voter/src/main/java/org/terracotta/voter/ActiveVoter.java

```diff
@@ -60,14 +57,19 @@
 
   private final Thread voter;
   private final String id;
-  private final String[] servers;
 
   private volatile boolean active = false;
+  private volatile ClientVoterManager activeClientVoterManager = null;
+  private volatile List<ClientVoterManager> voterManagers;
+  private volatile String existingTopology;
+  private volatile Set<String> registrationLatch;
```

Comment:
final rather than volatile

Reply:
activeClientVoterManager and existingTopology can't be final but yes others can be changed to final.